docs/configuration.md

```diff
@@ -236,6 +236,36 @@ CALICO_IPV6POOL_CIDR: "{{ spec.network.dualStack.IPv6podCIDR }}"
 | ---------------- | ------------------------------------------------------------------------------------------------ |
 | `disabled`       | Disable kube-proxy altogether (default: `false`).                                                |
 | `mode`           | Kube proxy operating mode, supported modes `iptables`, `ipvs`, `userspace` (default: `iptables`) |
+| Element    | Description                                                                                      |
+|------------|--------------------------------------------------------------------------------------------------|
```

Comment:
This looks bit wrong as it duplicates the headers

Reply:
Yeah, fixed, thanks!